_pages/oidc/authorization.md

```diff
@@ -130,6 +130,10 @@ In an **unsuccessful authorization**, the URI will contain the parameters `error
       <p>
         Multiple values can be joined with a space (before being URI-escaped in the final URL).
       </p>
+      <p>
```

Comment:
maybe in another pass we can go add `markdown="1"` to the `<div>` surrounding this so we can author text changes like this in markdown